app/assets/stylesheets/vendor/main.css

```diff
@@ -310,12 +310,6 @@ h4 span { font-family: 'source_sans_pro_semiboldRg', arial, sans-serif; }
 
 a:hover { text-decoration: underline; color: #2b2b2b; }
 
-.message {
-    display: inline-block;
-    border: 1px solid #bebebe;
-    padding: .3em .5em;
-    margin: .3em 1em 1em 0 !important;
-}
```

Comment:
What about in the [`search_results` partial](https://github.com/dpla/frontend/blob/develop/app/views/shared/results/_search_results.html.haml#L12-L17)?